envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -33,25 +34,30 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
             .readTimeout(Duration.ofSeconds(20))
             .build()
 
-        lateinit var envoyContainer: EnvoyContainer
+        lateinit var envoyContainer1: EnvoyContainer
+        lateinit var envoyContainer2: EnvoyContainer
+        lateinit var proxyContainer: ProxyContainer
```

Comment:
Why do we need another container? Could we use existing `localServiceContainer` instead? Or at least reuse that docker image? I don't see we use specific proxy features (calling with `?call=` parameter). But maybe I am missing something.

Reply:
You are right, first version used `call` to call another service but then I figured out it's not needed. Good catch, will remove :)